### DISCUSSION TITLE
DNS error when setting static IP

### DISCUSSION THREAD

**Discussion by Grek303** [esp8266/Arduino#8362] [General]
При использовании библиотек <ESP8266WiFi.h>, <WiFiUdp.h> и установке статического ip адреса не получается синхронизовать время из интернет. В чем может быть проблема???
[udp.txt](https://github.com/esp8266/Arduino/files/7465629/udp.txt)

**Comment by d-a-v** [comment]
> `WiFi.config(ip_cl, gateway_cl, subnet_cl);`

A DNS server address is missing, `europe.pool.ntp.org` can't be resolved without a DNS server.

Try with
`WiFi.config(ip_cl, gateway_cl, subnet_cl, gateway_cl);`
or 
`WiFi.config(ip_cl, gateway_cl, subnet_cl, IPAddress(8,8,8,8));`

**Reply by Grek303** [reply]
Thanks! This line helped: WiFi.config(ip_cl, gateway_cl, subnet_cl, gateway_cl);
